api/v1/clusterextensionrevision_types.go

```diff
@@ -55,9 +55,9 @@ type ClusterExtensionRevisionSpec struct {
 	// a unique value assigned. It is immutable after creation. The new revision number must always be previous revision +1.
 	//
 	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:Minimum:=1
+	// +kubebuilder:validation:Minimum=1
```

Comment:
The version with `:=` is documented: https://book.kubebuilder.io/reference/markers/crd-validation

You can also see that this is a no-op change - check the generated CRD.

Reply:
Ah, I didn't know that. Is there value in using a single format in the CRDs we produce?